pkg/proxy/ipvs/util/testing/fake.go

```diff
@@ -53,6 +54,19 @@ func (r *RealServerKey) String() string {
 	return net.JoinHostPort(r.Address.String(), strconv.Itoa(int(r.Port)))
 }
 
+// Implement https://pkg.go.dev/sort#Interface
+type ByAddress []*utilipvs.RealServer
```

Comment:
nitpick: we don't need to export `ByAddress`, right ?
maybe `byAddress` instead of `ByAddress`

Reply:
Right. Copy-paste syndrome :roll_eyes: